lib/backend/k8s/k8s_test.go

```diff
@@ -492,6 +506,114 @@ var _ = testutils.E2eDatastoreDescribe("Test Syncer API for Kubernetes backend",
 		})
 	})
 
+	It("should handle the static default-allow Profile", func() {
+
+		findAllowAllProfileEvent := func(c <-chan api.WatchEvent) bool {
+			found := false
+			for i := 0; i < 10; i++ {
+				select {
+				case e := <-c:
+					if e.Type == api.WatchAdded &&
+						e.Old == nil &&
+						e.New.Key == defaultAllowProfileKey {
+						found = true
+					}
+				default:
+					time.Sleep(50 * time.Millisecond)
+				}
+			}
+			return found
+		}
+
+		expectNoAllowAllEvent := func(c <-chan api.WatchEvent) {
+			found := findAllowAllProfileEvent(c)
+			Expect(found).To(BeFalse())
+		}
+
+		By("existing in our cache", func() {
+			expectedName := "projectcalico-default-allow"
+			Eventually(cb.GetSyncerValuePresentFunc(model.ProfileRulesKey{ProfileKey: model.ProfileKey{expectedName}}), slowCheck...).Should(BeTrue())
+		})
+
+		By("watching all profiles with any rv does not return an event for the default-allow profile", func() {
+			rvs := []string{"", "0"}
```

Comment:
The test says "with any rv" but we only use empty string and 0 here. Should probably throw in a few other RVs as well.

Reply:
152bb5f3bd7fa2fbec132f2b9ab5acd985112c13

Reply:
That commit seems to show you added other values, but this diff still shows only `""` and `"0"`?

Reply:
There are also a bunch of tests below this that also say "any rv" that should be updated to include more than these two values.